pkgs/top-level/linux-kernels.nix

```diff
@@ -346,6 +346,8 @@ in {
     nvidia_x11_vulkan_beta = nvidiaPackages.vulkan_beta;
     nvidia_x11             = nvidiaPackages.stable;
 
+    nvidia_x11_beta_open   = nvidiaPackages.beta.open;
```

Comment:
A comment on why it's there and that it's contents are not a replacement for `nvidia_x11`, but just the `.ko` files